includes/classes/SupportMonitor/Monitor.php

```diff
@@ -461,6 +471,7 @@ public function send_daily_report() {
 			$this->format_message(
 				[
 					'php_version' => $this->get_php_version(),
+					'db_queries'  => $this->db_query_monitor->get_report(),
```

Comment:
@felipeelia Do you know what size the data we can expect with 'db_queries'? My only concern is sending so much data that it pushes the limits of what can be sent through a message (and subsequently stored on the other database). I'd love to see some comments about expectations of what is sent and how we can make sure we're limiting it to a sane amount of data in the case there were 1000 alter statements called with a runaway script.

Reply:
@bengreeley the size of the data is a decision that still needs to be made. The 1,000 alter statements are somewhat related to the counter you asked in the other comment, by the way. If that is the same query, it would just have the count increased.

Reply:
With the new code, it is simply sending if heavy db queries were detected or not. The queries are only available on the site, not in Support Monitor anymore.